index.js

```diff
@@ -1,8 +1,9 @@
-var defined = require('defined');
 var createDefaultStream = require('./lib/default_stream');
 var Test = require('./lib/test');
 var createResult = require('./lib/results');
+var defined = require('./lib/util').defined;
 var through = require('through');
+var _ = require('underscore');
```

Comment:
i absolutely do not want to pull in underscore - even if this was an improvement, we'd want lodash over underscore a thousand times over.